app/controllers/tasks_controller.rb

```diff
@@ -22,7 +28,7 @@ def new
   end
 
   def create
-    @task = Task.new(create_params[:task])
+    @task = Task.new(create_params[:task]) #how to check for nil values?
```

Comment:
Per your comment on this line - since we've learned about them since this project, hopefully you can see how to to use model validations to check for nil now